index.js

```diff
@@ -220,23 +220,38 @@ function fail(msg) {
 }
 
 function showHistory(friendName) {
+  if (friendName == "--chronological") {
+    return showChronologicalHistory();
+  }
   var events = loadEventsData();
   var eventsByFriend = _.groupBy(events, "name");
-  Object.keys(eventsByFriend).sort().forEach(friend => {
-    if (!friendName || friendName === friend) {
-      prettyPrintFriendHeader(friend);
-      _.sortBy(eventsByFriend[friend], "date").reverse().map(prettyPrintEvent);
-      console.log();
-    }
-  });
+  var perFriend = function(friend) {
+    prettyPrintFriendHeader(friend);
+    _.sortBy(eventsByFriend[friend], "date").reverse().map(prettyPrintEventNameless);
```

Comment:
What changed here?

Reply:
Kunal were you looking at the full diff (not the diff for the individual commit)? The diff for the commit looks pretty clear.